vendor/assets/javascripts/changed_form_modal.js

```diff
@@ -0,0 +1,18 @@
+var NR = Cmsify.NestedResource.prototype;
+var CFM = Cmsify.ChangedFormModal = function(el) {
+  this.$el = $(el);
+  this.form = this.$el.find('form');
+  this.modal = UIkit.modal(this.$el.find('#js-unsaved-modal'));
```

Comment:
where is the ```#js-unsaved-modal``` coming from in the dom? Also, use a class for dom targeting with js